storage/src/cache/worker.rs

```diff
@@ -99,11 +99,14 @@ impl AsyncWorkerMgr {
         metrics: Arc<BlobcacheMetrics>,
         prefetch_config: Arc<AsyncPrefetchConfig>,
     ) -> Result<Self> {
-        // If the given value is less than maximum blob chunk size, it exceeds burst size of the
+        // If the given value is less than default blob chunk size, it exceeds burst size of the
         // limiter ending up with throttling all throughput, so ensure bandwidth is bigger than
-        // the maximum chunk size.
+        // the default chunk size.
         let tweaked_bw_limit = if prefetch_config.bandwidth_rate != 0 {
-            std::cmp::max(RAFS_MAX_CHUNK_SIZE as u32, prefetch_config.bandwidth_rate)
+            std::cmp::max(
+                RAFS_DEFAULT_CHUNK_SIZE as u32,
```

Comment:
Should this be RAFS_MAX_CHUNK_SIZE otherwise bigger chunks may be blocked for ever?

Reply:
Fixed